packages/shared/src/utils/config/config.ts

```diff
@@ -43,10 +43,10 @@ export const isTestLink = () => {
 };
 
 export const getAppId = () => {
-    let app_id = null;
+    let app_id: number | string | null = null;
     const user_app_id = ''; // you can insert Application ID of your registered application here
     const config_app_id = window.localStorage.getItem('config.app_id');
-    const current_domain = getCurrentProductionDomain();
+    const current_domain: string = getCurrentProductionDomain() || '';
     const platform = new URLSearchParams(window.location.search).get('platform');
 
     // Added platform at the top since this should take precedence over the config_app_id
```

Comment:
```suggestion
    // Added platform at the top since this should take precedence over the config_app_id
    if (platform && platform_app_ids[platform as keyof typeof platform_app_ids]) {
        app_id = platform_app_ids[platform as keyof typeof platform_app_ids];
```
@niloo-fs we need it to satisfy TS here as it's still giving an error here